deid/dicom/utils.py

```diff
@@ -280,3 +280,9 @@ def get_item_timestamp(dicom,date_field=None,time_field=None):
 
     return get_timestamp(item_date=item_date,
                          item_time=item_time)
+
+def setup_deid(deid):
+    # if the user has provided a custom deid, load it
```

Comment:
Please add a documentation snippet to the function, and do y think we should support loading a base (default does not) and optionally taking a list of then to combine? Is setup the most appropriate descriptor for the function ?

Reply:
@vsoch I couldn't come up with a better name, but I don't think it's the most appropriate. I also don't like passing deid around with multiple meanings seems confusing to me when it sometimes refers to a file path and other times refers to the object.

Reply:
I completely agree - let me give a first shot at making a class and I'll ping you for review. A class would be important to avoid needing to re-generate the final thing, and also to (eventually) plug into generators and editors. Back in a bit!